Vagrantfile.freebsd

```diff
@@ -0,0 +1,64 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+#   Copyright The containerd Authors.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# Vagrantfile for FreeBSD
+Vagrant.configure("2") do |config|
+  config.vm.box = "generic/freebsd13"
+
+  memory = 2048
+  cpus = 1
```

Comment:
do we need two cpus here?

Reply:
Why?

Reply:
Because I see the original one (.cirrus.yml) was using two cpus 😂